editor/src/uuiui/inputs/number-input.tsx

```diff
@@ -977,13 +969,14 @@ interface ChainedNumberControlProps {
   idPrefix: string
   style?: React.CSSProperties
   setGlobalCursor?: (cursor: CSSCursor | null) => void
+  wrap?: boolean
 }
 
 export const ChainedNumberInput: React.FunctionComponent<
   React.PropsWithChildren<ChainedNumberControlProps>
-> = React.memo(({ propsArray, idPrefix, style, setGlobalCursor }) => {
+> = React.memo(({ propsArray, idPrefix, style, setGlobalCursor, wrap }) => {
   return (
-    <FlexRow style={style}>
+    <FlexRow style={{ ...style, flexWrap: wrap ? 'wrap' : 'nowrap' }}>
```

Comment:
very minor - but better to do `{{ flexWrap: wrap ? 'wrap' : 'nowrap', ...style }}` so that the consumer will be able to override (for example if they pass `style={{ flexWrap: 'wrap'}}` but no `wrap` prop